utils/utils.go

```diff
@@ -142,6 +143,12 @@ func CustomizeLibertyEnv(pts *corev1.PodTemplateSpec, la *wlv1.WebSphereLibertyA
 		)
 	}
 
+	if ba.GetManageTLS() == nil || *ba.GetManageTLS() {
+		if _, found := findEnvVar("SEC_IMPORT_K8S_CERTS", pts.Spec.Containers[0].Env); !found {
```

Comment:
The env we want to add can simply be added to targetEnv and the env check below would skip if it's set by user.